bluebottle/members/tests/test_admin.py

```diff
@@ -301,13 +301,15 @@ class MemberPlatformSettingsAdminTestCase(BluebottleAdminTestCase):
 
     def test_require_location(self):
         self.app.set_user(self.superuser)
+        user = BlueBottleUserFactory.create()
         page = self.app.get(reverse('admin:members_memberplatformsettings_change'))
         form = page.forms[0]
         form['require_office'].checked = True
 
         form.submit()
         settings_platform = MemberPlatformSettings.load()
-        self.assertTrue(settings_platform.require_office)
+        if user.place is None:
```

Comment:
Putting if statements in tests like this does not make any sense. Either `user.place` is always `None` in this test, and the line after this is never hit, or it is never `None` and this line is always hit. Test will, and should, always behave the same. If `if` statement would do something, sometimes the tests might fail, and sometimes they will not. That would be really annoying.